my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
```

Comment:
on the filename: try to use more descriptive filenames; `my_code.py` is not very descriptive